### PR TITLE
First Name and Last Name in Register

### PR DESCRIPTION
**Pull request by ShrlGnwn**
Link Card : https://trello.controlpanel.gg/b/KGMuFqkqe5QabLt76/controlpanel/qLZwNk6xbKbLbspqw

image
image
image

### CODE REVIEW

**Comment by AGuyNamedJens** [comment]
Has this been tested and is it complete?

**Comment by ShrlGnwn** [comment]
> Has this been tested and is it complete?

this has been tested, and should be complete

File: app/Http/Controllers/Auth/RegisterController.php [hunk 1]
```diff
@@ -147,13 +149,20 @@ protected function validatePterodactylUser(string $email)
      */
     protected function createPterodactylUser(Request $request, User $user): array
     {
+        //check last_name
```
Comment (AVMG20) [inline app/Http/Controllers/Auth/RegisterController.php:152]: //check last_name
```php 
$last_name = empty($user->last_name) ? $user->first_name : $user->last_name;
```

File: database/seeders/Standard/NotificationSeeder.php [hunk 2]
```diff
@@ -20,7 +20,7 @@ public function run()
             'name' => 'welcome-message',
             'title' => "Getting started",
             'content' => "
-               <p>Hello <strong>{{\$user->name}}</strong>, Welcome to our dashboard!</p>
+               <p>Hello <strong>{{\$user->username}}</strong>, Welcome to our dashboard!</p>
```
Comment (AVMG20) [inline database/seeders/Standard/NotificationSeeder.php:23]: Good one! :+1:

**Review by AVMG20 — APPROVED** [review]

**Pull request merged**